### ISSUE TITLE
Link error in Gigasecond

### ISSUE THREAD

**Issue by blakelewis** [exercism/cpp#514]
Hi, I recently discovered Exercism and have become somewhat addicted.  I wasn't able to finish the C++ Gigasecond exercise because it can't find the required Boost library:

make[2]: *** No rule to make target '/usr/lib/libboost_date_time.a', needed by 'gigasecond'.  Stop.
make[1]: *** [CMakeFiles/Makefile2:77: CMakeFiles/gigasecond.dir/all] Error 2
make: *** [Makefile:84: all] Error 2

I am working in the browser editor.

I  skipped Meetup also because it depends on Boost.  Wouldn't it make sense to use  the date and time functions in the standard library instead of Boost?

**Comment by patricksjackson** [comment]
I'm trying to sort this out in https://github.com/exercism/exercism/issues/6332. I'm not sure what's going on, but the exercise should work in the online editor since #495 was checked in back in December. I've confirmed it works starting from a new account, but maybe you downloaded the gigasecond exercise before last December?

The exercises use Boost because we haven't updated the repo to C++20, where the date time utilities in the stdlib provide a decent interface. And because I didn't want to bother copying over the C tracks solutions here when I ripped Boost out of the rest of the track, since the solutions would be identical.

Considering complaints about Boost are still the largest single source of issue with the track, we should probably try to do that sooner rather than later.

The Boost dependency is 100% of the reason why these exercises are marked as optional.
